Overview.src.html

```diff
@@ -2306,8 +2329,8 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 
 <h3>Main fetch</h3>
 
-<p>To perform a <dfn title=concept-main-fetch>main fetch</dfn> using <var>request</var>, optionally
-with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
+<p>To perform a <dfn title=concept-main-fetch data-export>main fetch</dfn> using <var>request</var>,
+optionally with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
```

Comment:
I guess it's fine to export in that case. Hopefully nobody ever considers implementing a public API for these.